packages/core/lib/commands/test/run.js

```diff
@@ -38,16 +38,21 @@ module.exports = async function (options) {
   }
 
   // Start managed ganache network
-  async function startGanacheAndRunTests(ipcOptions, ganacheOptions, config) {
+  async function startGanacheAndRunTests(
+    ipcOptions,
+    ganacheOptions,
+    truffleConfig
```

Comment:
why change this variable to `truffleConfig`? we don't really have a convention of doing this

Reply:
It made the code less confusing for me. Since `startGanacheAndRunTests` only uses positional arguments, the renaming won't impact the callers.

Reply:
Well I'm not going to fight you on it, but we use `config` like literally everywhere else in the codebase.